src/test/java/org/broadinstitute/hellbender/utils/python/PythonScriptExecutorUnitTest.java

```diff
@@ -0,0 +1,124 @@
+package org.broadinstitute.hellbender.utils.python;
+
+import org.apache.commons.io.FileUtils;
+import org.broadinstitute.hellbender.utils.io.IOUtils;
+import org.broadinstitute.hellbender.utils.io.Resource;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public class PythonScriptExecutorUnitTest extends BaseTest {
+    final String HELLO_WORLD_SCRIPT = "print \"hello, world\"";
+
+    @Test(groups = {"Python"})
```

Comment:
I think all of our other groups are lowercase maybe this one should be too?

Reply:
Done.